navpy/core/navpy.py

```diff
@@ -41,6 +41,12 @@ def angle2dcm(rotAngle1, rotAngle2, rotAngle3, input_unit='rad',
     Returns
     --------
     C : {3x3} Direction Cosine Matrix
+        
+    Examples
+    --------
+    >>> import navpy
+    >>> # Returns the direction matrix for a yaw rotation of 90 degrees (0,0,90)
+    >>> dcm_matrix = navpy.angle2dcm(0, 0, 90, input_unit="deg")
```

Comment:
Can the description be added above the example?  The style guide for numpy can serve as a reference: https://numpydoc.readthedocs.io/en/latest/format.html#examples

The examples should also include the printout, so that the user can check if they get the same answer (or make sense of it without running it).